core/tasks/model_monitoring.py

```diff
@@ -156,4 +156,5 @@ def calculate_model_metrics():
         feature_drift_df = feature_drift.compute_feature_drift(
             ref_n_samples=10, cur_n_samples=10
         )
```

Comment:
@Rup-Narayan-Rajbanshi Why only 10 samples ? We need to pass all the samples collected over a period of time (e.g. week or daily -> depends upon when the task is scheduled).

Reply:
use like this: feature_drift.comptue_feature_drift(len(reference_data_df), len(current_data_df))